packages/shared/components/ColoredCircle.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+    export let color: string
+    export let size: number = 3
+</script>
+
+<colored-circle class="self-center rounded-full w-{size} h-{size}" style:--color={color} />
```

Comment:
I don't know if `w-{size} h-{size}` would be the best approach because . Maybe a `style:--size="{size}px"` var would be better, or an enum with `Small, Medium, Large, ...` values